src/components/home/ThumnailArea.css

```
@@ -0,0 +1,21 @@
+.thumnail__list__container{
+    display: flex;
+    justify-content: space-around;
+}
+@media (max-width: 1440px){
+}
+/* 화면 너비 0 ~ 1200px */
+@media (max-width: 1220px){
+}
+/* 화면 너비 0 ~ 768px */
+@media (max-width: 768px){
+    .thumnail__list__container{
+        width: auto;
+        display: inline-flex;
+        padding: 0 24px;
+    }
+}
```

```suggestion
@mixin mobile {
  @media (min-width: #{$breakpoint-mobile}) and (max-width: #{$breakpoint-tablet - 1px}) {
    @content;
  }
}

```

https://chaewonkong.github.io/posts/scss-media-query.html 
scss를 활용해서 조금 더 반응형을 이쁘게 대응하는건 어떨까요?